api/base/filters.py

```diff
@@ -32,7 +32,42 @@ def intersect(x, y):
     return x & y
 
 
-class ODMFilterMixin(object):
+class FilterMixin(object):
+    """ View mixin with helper functions for filtering. """
+
+    TRUTHY = set(['true', 'True', 1, '1'])
+    FALSY = set(['false', 'False', 0, '0'])
+    DEFAULT_OPERATOR = 'eq'
+
+    def is_filterable_field(self, key):
+        try:
+            return key.strip() in self.serializer_class.filterable_fields
```

Comment:
I don't like how ```self.serializer_class``` is just expected (guessing this comes from elsewhere in the inheritance hierarchy).

Consider adding something like:
```
def __init__(self, *args, **kwargs):
    super(FilterMixin, self).__init__(*args, **args)
    if not self.serializer_class:
         raise NotImplementedError()
```
So that applying this mixin to a class without ```serializer_class``` will fail loudly.